src/main/webapp/app/exercises/shared/exercise/exercise-details/exercise-details.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { SafeHtml } from '@angular/platform-browser';
-import { Exercise, ExerciseType } from 'app/entities/exercise.model';
+import { Exercise, ExerciseType, getExerciseUrlSegment } from 'app/entities/exercise.model';
```

Comment:
```suggestion
import { Exercise, ExerciseType } from 'app/entities/exercise.model';
```
probably not needed when log is gone